File: snake-server/Person.py
class coffee.Person:
    def __init__(self, name=None, hobbies=None):
        self.name = name
        self.hobbies = hobbies if hobbies is not None else []

    def __str__(self):
        return f"coffee.Person(name={self.name}, hobbies={self.hobbies})"

File: coffee-server/src/main/java/coffee/Person.java
package coffee;

import com.thoughtworks.xstream.annotations.XStreamAlias;
import com.thoughtworks.xstream.annotations.XStreamImplicit;
import lombok.*;

import java.util.List;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@ToString
@XStreamAlias("Person")
public class Person {

    private String name;

    @XStreamImplicit(itemFieldName = "hobby")
    private List<String> hobbies;
}
